python/setup.py

```diff
@@ -13,7 +14,7 @@
 
 # extras
 dev = [
-    "black",
+    "black==21.11b1",
```

Comment:
the last stable one is 22.10?

https://pypi.org/project/black/

Reply:
It makes sense, I will change to `22.1.0`